README.md

```diff
@@ -2,13 +2,37 @@
 [![](https://img.shields.io/badge/installations-Prerequisities-important)](https://github.com/IIITM-Jay/Property-management-System-with-multiple-layers-Django-Full-Stack-/blob/main/requirements.txt)
 [![](https://img.shields.io/badge/download-project%20report-success)](https://drive.google.com/file/d/1fYfPtsnYsmvmAFY8uaJ9S8ApwlAV5MZY/view?usp=sharing)
 [![](https://img.shields.io/badge/basics-of%20django-brightgreen)](https://www.djangoproject.com/)
+<div align="center">
+<table>
+<thead>
+  <tr>
+    <th>Test</th>
+    <th>Status</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Github build</td>
+    <td><img src="https://github.com/IIITM-Jay/Property-management-System-Django-Full-Stack-/workflows/CI/badge.svg"></td>
+  </tr>
+  <tr>
+    <td>CodeQL</td>
+    <td><img src="https://github.com/IIITM-Jay/Property-management-System-Django-Full-Stack-/workflows/CodeQL/badge.svg"></td>
+  </tr>
+  <tr>
+    <td>Manual Worklow</td>
+    <td><img src="https://github.com/IIITM-Jay/Property-management-System-Django-Full-Stack-/workflows/Manual%20workflow/badge.svg"></td>
+  </tr>
+  <tr>
+    <td>New Contributors</td>
+    <td><img src="https://github.com/IIITM-Jay/Property-management-System-Django-Full-Stack-/workflows/Greetings/badge.svg"></td>
+  </tr>
+</tbody>
+</table>
+</div>
+
```

Comment:
@himanshu007-creator Himanshu, this is simply an HTML code, can you try using the markdown format as it is the standard format for writing the readme files.